content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -998,6 +1006,14 @@ The migrations from the database are not found locally in prisma/migrations:
 20201208100950_new_migration
 ```
 
+In versions 4.3.0 and later, `prisma migrate status` exits with exit code 1 in the following cases:
+
+- a database connection error occurs
+- there are migration files that have not been applied to the database
+- the migration history has diverged
```

Comment:
Just checking, diverged from what? The state of the database, or the migration table, or something else?

Reply:
Between the migration history (that is calculated from the `./migrations` directory and the database)
I think many cases can happen here, any difference between the directory and the database state = fail.

Reply:
Thanks, added some words to clarify